src/CMakeLists.txt

```diff
@@ -184,11 +184,13 @@ endif()
 set(valhalla_hdrs
     ${VALHALLA_SOURCE_DIR}/valhalla/valhalla.h
     ${VALHALLA_SOURCE_DIR}/valhalla/worker.h
+    ${VALHALLA_SOURCE_DIR}/valhalla/config.h
```

Comment:
perfect, just did that in another PR as well, was an oversight in the initial one